src/clients/python/crequest.cc

```diff
@@ -724,6 +733,8 @@ InferContextAsyncResultNew(
 {
   InferContextResultCtx* lctx = new InferContextResultCtx;
```

Comment:
Not related to this change, but this should be managed by smart pointer in case of early return.

Reply:
Fixed by moving the creation till after the early returns.